services/ark-dashboard/ark-dashboard/components/app-sidebar.tsx

```diff
@@ -55,23 +56,71 @@ export function AppSidebar() {
     const loadInitialData = async () => {
       setLoading(true)
       try {
+        // Load critical data first (namespaces and system info)
         const [namespacesData, systemData] = await Promise.all([
           namespacesService.getAll(),
           systemInfoService.get()
         ])
+        
         const filteredNamespaces = namespacesData.filter(ns => 
           !['cert-manager', 'kube-node-lease', 'kube-system', 'kube-public'].includes(ns.name)
         )
         setNamespaces(filteredNamespaces)
         setSystemInfo(systemData)
         setNamespaceResolved(true)
         
-        // Read namespace from URL if present
-        const urlParams = new URLSearchParams(window.location.search)
-        const urlNamespace = urlParams.get('namespace')
-        if (urlNamespace && filteredNamespaces.some(ns => ns.name === urlNamespace)) {
-          setNamespace(urlNamespace)
+        // Try to get current namespace and mode information
+        let selectedNamespace = "default"
```

Comment:
once we've made the described changes on startup we can just set the url param once to the current context namespace and it'll then simply continue to use that, no need for single/multi mode, we just only show one namespace in the drop down box (then later for cluster admins we can bring back the other namespaces)